### ISSUE TITLE
Broken Page

### ISSUE THREAD

**Issue by KenLWilson** [Compass/compass#2067]
On Page: http://compass-style.org/help/tutorials/configuration-reference/

Dead Link

**Comment by JamesVanWaza** [comment]
@KenLWilson, Compass is no longer supported by https://twitter.com/chriseppstein

**Comment by KenLWilson** [comment]
@jgacuca567 can you please elaborate? Not supported by the creators or on GitHub? Is it not worth learning/using if I'm new to it?

**Comment by JamesVanWaza** [comment]
@KenLWilson , yes its not supported by the Creator anymore, use either Libsass or Eyeglass.
